scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/ActivityCreationTemplate.java

```diff
@@ -108,7 +106,6 @@ public Builder basedOn(ActivityCreationTemplate template) {
       endsIn = template.endRange;
       durationIn = template.durationRange;
       startsOrEndsIn = template.startOrEndRange;
-      nameMatches = (template.nameRE != null) ? template.nameRE.pattern() : null;
```

Comment:
it has been moved to commit #d113e2ea

Reply:
I am confused, because the commit you shared @adrienmaillard is _this_ commit. Was the change in a different commit and you moved it to this one? Perhaps Github moved this conversation to follow the change, if so?